### PR TITLE
user inactive can not log in

### PR DESCRIPTION
**Pull request by neliojrr**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -236,6 +236,16 @@ def send_devise_notification(notification, *args)
     devise_mailer.send(notification, self, *args).deliver_later
   end
 
+  # check if user is active or not
+  def active_for_authentication?
+    super and self.active?
```
Comment (CGA1123) [inline app/models/user.rb]: Travis is complaining about using `and`, `&&` is preffered!

**Comment by CGA1123** [comment]
Seems to work good. Deals with the reset password loophole as well.

File: app/models/user.rb [hunk 2]
```diff
@@ -236,6 +236,16 @@ def send_devise_notification(notification, *args)
...
   end
 
+  # check if user is active or not
+  def active_for_authentication?
+    super && self.active?
+  end
+
+  # message to the user that is not allowed to login
+  def inactive_message
+    "You are not allowed to log in!"
```
Comment (muescha) [inline app/models/user.rb]: missing i18n

Comment (lccezinha) [inline app/models/user.rb]: The code is simple enough. It's not necessary to comment it.

Comment (scott) [inline app/models/user.rb]: I think @muescha is saying to use an i18n string instead of hard coding the message

Comment (muescha) [inline app/models/user.rb]: but @lccezinha point to an other problem: to reduce comments for just one liners. because the often get out of sync later

Comment (scott) [inline app/models/user.rb]: Oh yeah thats true @muescha I didn't even pay attention to the comment

**Comment by scott** [comment]
Yeah seems to work well, thanks for helping on this everyone!

**Comment by muescha** [comment]
checked in without i18n :(

**Comment by scott** [comment]
Submit a PR :)  Its an important bug fix that has been waiting for a long time or I wouldn't have.

**Pull request merged**